packages/nestjs-user/src/user.utils.ts

```diff
@@ -0,0 +1,16 @@
+import { PasswordStrengthEnum } from '@concepta/nestjs-password';
+import { PasswordStrengthByRoleCallback } from './user.types';
+
+export const defaultPasswordStrengthByRole: PasswordStrengthByRoleCallback = (
```

Comment:
i made a comment somewhere else, where this can receive many different context in one object, being specific to roles is probably too narrow.